### PR TITLE
formly dialog initial

### PR DESCRIPTION
**Pull request by ajaybal0082**
## Type of Change (Select One and Apply Github Label)

- [ ] Bug fix (non-breaking change which fixes an issue) -> Apply bugfix label
- [ ] New feature (non-breaking change which adds functionality) -> Apply enhancement label
- [ ] Breaking change (fix or feature that would cause existing functionality to change) -> Apply breaking label

## Which browsers have you tested?
- [ ] Internet Explorer 11
- [ ] Edge
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

## Checklist:

- [ ] I have read the (https://github.comv/GSA/sam-ui-elements/blob/CONTRIBUTING.md)[CONTRIBUTING.md] document.
- [ ] My code passes the automated linter.
- [ ] This code has been reviewed by another team member and passes the reviewer checklist found in (https://github.comv/GSA/sam-ui-elements/blob/CONTRIBUTING.md)[CONTRIBUTING.md]
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] My code is 508 compliant as tested by AMP and JAWS
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: libs/packages/sam-formly/src/lib/formly-filters/sds-filters.component.ts [hunk 1]
```diff
@@ -8,11 +8,43 @@ import { FormlyFieldConfig } from '@ngx-formly/core';
...
 import { SDSFormlyUpdateComunicationService } from './service/sds-filters-comunication.service';
 import { pairwise } from 'rxjs/operators';
+import {
+  SdsDialogService,
+  SdsDialogRef,
+  SDS_DIALOG_DATA
+} from '@gsa-sam/components';
+export interface DialogData {
+  keyword: boolean;
+  title: boolean;
```
Comment (yerramshilpa) [inline libs/packages/sam-formly/src/lib/formly-filters/sds-filters.component.ts]: why are you adding hardcoding values

File: libs/packages/sam-formly/src/lib/formly-filters/sds-filters.module.ts [hunk 2]
```diff
@@ -9,11 +9,15 @@ import {
...
 import { SdsFiltersComponent } from './sds-filters.component';
 import { SdsFormlyModule } from '../formly/formly.module';
-
+import { SdsDialogModule } from '@gsa-sam/components';
+import { SDS_DIALOG_SCROLL_STRATEGY_PROVIDER, SdsDialogService  } from '@gsa-sam/components';
+import{ DialogAdvancedFilterDialog } from './sds-filters.component';
 @NgModule({
     declarations: [
         SdsFiltersComponent,
+        DialogAdvancedFilterDialog
```
Comment (yerramshilpa) [inline libs/packages/sam-formly/src/lib/formly-filters/sds-filters.module.ts]: delete duplicate dialog word rename something like SdsAdvanceFilterDialog

File: libs/packages/sam-formly/src/lib/formly-filters/sds-more-filters.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
+<div sds-dialog-title>Advanced Filters</div>
+
```
Comment (yerramshilpa) [inline libs/packages/sam-formly/src/lib/formly-filters/sds-more-filters.html]: Can you make dialog title configurable

File: libs/sam-formly-sample/src/lib/feature/formly-dialog/formly-advanced-filters-dialog.html [hunk 4]
```diff
@@ -0,0 +1,60 @@
+<div sds-dialog-title>Advanced Filters</div>
```
Comment (yerramshilpa) [inline libs/sam-formly-sample/src/lib/feature/formly-dialog/formly-advanced-filters-dialog.html]: what is this dialog..?

File: libs/sam-formly-sample/src/lib/feature/formly-dialog/formly-dialog.component.ts [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+import {
+    SdsDialogService,
+    SdsDialogRef,
+    SDS_DIALOG_DATA
+  } from '@gsa-sam/components';
+import { removeSummaryDuplicates } from '@angular/compiler';
+import { text } from '@angular/core/src/render3';
+import { _MatButtonMixinBase } from '@angular/material/button';
+  export interface DialogData {
+    keyword: boolean;
```
Comment (yerramshilpa) [inline libs/sam-formly-sample/src/lib/feature/formly-dialog/formly-dialog.component.ts]: what is this data for..?

File: libs/sam-formly-sample/src/lib/feature/formly-dialog/formly-advanced-filters-dialog.html (hunk 4 above)
Comment (ajaybal0082) [reply libs/sam-formly-sample/src/lib/feature/formly-dialog/formly-advanced-filters-dialog.html]: This was a temporary file and I have removed it now

File: libs/sam-formly-sample/src/lib/feature/formly-dialog/formly-dialog.component.ts (hunk 5 above)
Comment (ajaybal0082) [reply libs/sam-formly-sample/src/lib/feature/formly-dialog/formly-dialog.component.ts]: This is not required anymore...I have made the changes and removed it

File: libs/packages/sam-formly/src/lib/formly-filters/sds-filters.component.ts (hunk 1 above)
Comment (ajaybal0082) [reply libs/packages/sam-formly/src/lib/formly-filters/sds-filters.component.ts]: that has been removed

File: libs/packages/sam-formly/src/lib/formly-filters/sds-filters.module.ts (hunk 2 above)
Comment (ajaybal0082) [reply libs/packages/sam-formly/src/lib/formly-filters/sds-filters.module.ts]: done

File: libs/packages/sam-formly/src/lib/formly-filters/sds-more-filters.html (hunk 3 above)
Comment (ajaybal0082) [reply libs/packages/sam-formly/src/lib/formly-filters/sds-more-filters.html]: as per our discussion with Dave, we don't have to.

File: libs/packages/sam-formly/src/lib/formly-filters/sds-filters.component.ts [hunk 6]
```diff
@@ -13,24 +13,18 @@ import {
...
-  keyword: boolean;
-  title: boolean;
-  currentDate: boolean;
-  publishDate: boolean;
-  lastModifiedDate: boolean;
-  FederalFilter: boolean;
-  noticeFilter: boolean;
-  ExpirationDateFilter: boolean;
-}
+import { analyzeAndValidateNgModules } from '@angular/compiler';
```
Comment (yerramshilpa) [inline libs/packages/sam-formly/src/lib/formly-filters/sds-filters.component.ts:16]: remove unused import

**Pull request closed without merging**